pkg/controller/vitesscluster/reconcile_vtadmin.go

```diff
@@ -23,6 +23,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"path/filepath"
```

Comment:
nit: we should add a go-import check to that repository

Reply:
Yep, we should. I'll fix this for now

Reply:
Fixed